src/__tests__/typed-router.test.ts

```diff
@@ -45,9 +44,13 @@ test('TypedRouter', async () => {
   });
 
   router.post('/users', async (_, user, request, response) => {
-    assertType(user, _ as {age: number; name: string});
-    assertType(request.params, _ as {});
-    assertType(request.body, _ as {age: number; name: string});
+    const _userForCheck = user;
+    //     ^? const _userForCheck: CreateUserRequest
```

Comment:
There's a mismatch here between the type in quickinfo and what the eslint plugin wants:

<img width="718" alt="image" src="https://user-images.githubusercontent.com/98301/168474628-84b446e2-48ea-44ad-b610-95abb0c9fe00.png">